src/components/UI/ResourcesCard/ResourcesCard.tsx

```diff
@@ -0,0 +1,153 @@
+import styled from "@emotion/styled";
+import React, { FunctionComponent } from "react";
+import { Download } from "react-feather";
+import { mixin, vars } from "./../../../styles";
+
+interface ResourcesCardProps {
+  className?: string;
+  details: {
+    title: string;
+    youtubeEmbedCode?: string;
+    placeholderText?: string;
+    tag: string;
+    description: string;
+    downloads: {
+      fileName: string;
+      path: string;
+    }[];
+  };
+}
+
+const ResourcesCardUnStyled: FunctionComponent<ResourcesCardProps> = ({ className, details }) => {
+  const { placeholderText, youtubeEmbedCode, title, tag, description, downloads } = details;
+  const hasMedia = placeholderText || youtubeEmbedCode;
+
+  return (
+    <div className={`${className}`}>
+      {hasMedia && (
+        <div className="media-holder">
```

Comment:
the 16:9 trick did not work here (causing video to be cropped), u might wanna open another `row` and `col-12 col-lg-6` it will ensure that the video will not be cropped, but doing so will result in something this:

![Screenshot 2020-09-28 at 4 19 05 PM](https://user-images.githubusercontent.com/4774314/94407775-78ee1580-01a6-11eb-905b-261d8cc9a5b7.png)

this now will have the white bars on top and bottom which is not nice also so u might wanna somehow add black to the video part instead so it will look like this:

![Screenshot 2020-09-28 at 4 21 15 PM](https://user-images.githubusercontent.com/4774314/94407921-b3f04900-01a6-11eb-8165-28b3ed8bd671.png)